torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5744,7 +5744,7 @@ def sample_inputs_cross_entropy(op_info, device, dtype, requires_grad, **kwargs)
 
             if "ignore_index" in kwargs and torch.all(target == kwargs["ignore_index"]):
                 # make sure at least one item in target is not ignored
-                target[0] = random.sample(set(range(num_classes)) - {kwargs["ignore_index"]}, 1)[0]
+                target[0] = random.sample(sorted(set(range(num_classes)) - {kwargs["ignore_index"]}), 1)[0]
```

Comment:
why sorted versus list()? extra overhead and it doesn't buy you any randomness :P

Reply:
Because that's what the Python god recommended in the error message:
```
  File "/home/albandes/local/pytorch/3.11_debug_source/torch/testing/_internal/common_methods_invocations.py", line 5747, in sample_inputs_cross_entropy
    target[0] = random.sample(set(range(num_classes)) - {kwargs["ignore_index"]}, 1)[0]
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/albandes/local/installs/python3.11/debug/install/lib/python3.11/random.py", line 439, in sample
    raise TypeError("Population must be a sequence.  "
TypeError: Population must be a sequence.  For dicts or sets, use sorted(d).
```